felles/src/main/kotlin/no/nav/familie/kontrakter/felles/tilbakekreving/HentFagsystemsbehandlingRequest.kt

```diff
@@ -8,6 +8,7 @@ import javax.validation.constraints.Pattern
 data class HentFagsystemsbehandlingRequest(
     val eksternFagsakId: String,
     val ytelsestype: Ytelsestype,
+    val regelverk: Regelverk,
```

Comment:
Denne brukes nog av flere applikasjoner, så for å ikke brekke de applikasjonene som ikke har oppdatert kontrakter så burde den være nullable, eller default value.
Hvis ikke så kommer familie-tilbake kaste feil når eks familie-ef-iverksett kaller på tilbake uten å sende med regelvek

Reply:
Sjekk min kommentar over. :)